CHANGELOG.md

```diff
@@ -18,6 +18,7 @@
 - Expose wakeup via REST-API
 - Update database indices to match workflow instance polling code
 - Add new configuration option `nflow.db.disable_batch_updates`
+- Workarounds for Galera oddities to make nflow more stable
```

Comment:
```suggestion
- Workarounds for Galera oddities to make nFlow more stable when Galera is used
```

Reply:
I don't think we did anything except that above `nflow.db.disable_batch_updates` actually. If not documenting in Wiki counts?